frontend2/src/content/ContentStruct.ts

```diff
@@ -0,0 +1,57 @@
+export type QuickStartedPageKey =
+  | "overview"
+  | "creation"
+  | "installation"
+  | "resources"
+  | "community";
+export enum QuickStartedPage {
+  OVERVIEW = "overview",
+  ACCOUNT_AND_TEAM_CREATION = "creation",
+  INSTALLATION = "installation",
+  RESOURCES = "resources",
+  JOIN_THE_COMMUNITY = "community",
+}
+
+export type ResourcesPageKey =
+  | "specification"
+  | "resources"
+  | "tools"
+  | "lectures";
+export enum ResourcesPage {
+  GAME_SPECIFICATION = "specification",
+  CODING_RESOURCES = "resources",
+  THIRD_PARTY_TOOLS = "tools",
+  LECTURES = "lectures",
+}
```

Comment:
Did some digging on TS type tricks [here (link)](https://stackoverflow.com/questions/52393730/typescript-string-literal-union-type-from-enum/64966647#64966647). Looks like as of TS 4.1+, we can do something like this:
```
export type ResourcesPageKey = `${ResourcesPage}`;
```
I think this is a neat trick and will help DRY out this file :sunglasses: